tendrl/ceph_integration/objects/pool/atoms/update/__init__.py

```diff
@@ -0,0 +1,28 @@
+from tendrl.ceph_integration.manager.crud import Crud
+from tendrl.ceph_integration import objects
+from tendrl.ceph_integration.objects.pool import Pool
+
+
+class Update(objects.CephIntegrationBaseAtom):
+    obj = Pool
+    def __init__(self, *args, **kwargs):
+        super(Update, self).__init__(*args, **kwargs)
+
+    def run(self):
+        pool_id = self.parameters['Pool.pool_id']
+        attrs = {}
+        if 'Pool.poolname' in self.parameters:
+            attrs['name'] = self.parameters.get('Pool.poolname')
+        if 'Pool.pg_num' in self.parameters:
+            attrs['pg_num'] = self.parameters.get('Pool.pg_num')
```

Comment:
There is no specific check here as such. May be we can fetch the value from etcd and compare. If new value if smaller than existing one, we can raise an exception here.